src/coverlet.msbuild.tasks/MSBuildLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using ILogger = Coverlet.Core.Logging.ILogger;
+
+namespace Coverlet.MSbuild.Tasks
+{
+    class MSBuildLogger : ILogger
+    {
+        private readonly TaskLoggingHelper _log;
+
+        public MSBuildLogger(TaskLoggingHelper log) => _log = log;
+
+        public void LogVerbose(string message) => _log.LogMessage(message, MessageImportance.Low);
```

Comment:
I think this is wrong, should be ` _log.LogMessage(MessageImportance.Low, message);` https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.tasklogginghelper.logmessage?view=netframework-4.7.2&viewFallbackFrom=netcore-2.2#Microsoft_Build_Utilities_TaskLoggingHelper_LogMessage_Microsoft_Build_Framework_MessageImportance_System_String_System_Object___